templates/eap-cd-amq-persistent-s2i.json

```diff
@@ -65,9 +65,9 @@
         },
         {
             "displayName": "JMS Connection Factory JNDI Name",
-            "description": "JNDI name for connection factory used by applications to connect to the broker, e.g. java:/DefaultConnectionFactory",
+            "description": "JNDI name for connection factory used by applications to connect to the broker, e.g. java:jboss/DefaultJmsConnectionFactory",
             "name": "MQ_JNDI",
-            "value": "java:/DefaultConnectionFactory",
+            "value": "java:jboss/DefaultJmsConnectionFactory",
```

Comment:
@luck3y it is `java:jboss/DefaultJMSConnectionFactory`

Reply:
Sigh, sorry @tremes, thats what I get for trying to use sed before I've had enough coffee!